src/vs/vscode.d.ts

```diff
@@ -5447,14 +5452,17 @@ declare module 'vscode' {
 	export namespace scm {
 
 		/**
-		 * The [input box](#SourceControlInputBox) in the Source Control viewlet.
+		 * The [input box](#SourceControlInputBox) for the last source control
```

Comment:
Move `@deprecated` to the top and enclose the rest in `~~` to strike it out

Reply:
Maybe also explain the semantics of the 'old' deprecated `inputBox`, e.g last active one